docs/using/masks.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: ../links.inc
```

Comment:
Overall really cool, I think it will also be good to state here that in BOLD preprocessing of large 4D NIfTI files, the masks can have a beneficial effect on memory usage (just because when i started doing BOLD processing with nilearn without a mask, I would get the occasional memory error).

Reply:
I am no expert in fMRI analysis, that's why I don't want to add much. I think this section could be improved.

Also, if we "mask" at the preprocessing, then all the markers _must_ set the mask to "inherit". Otherwise, non-clean voxels might be used.